README.md

```diff
@@ -0,0 +1,144 @@
+$ kubectl delete -f https://raw.githubusercontent.com/skydive-project/skydive-operator/master/deploy/role_binding.yaml
+$ kubectl delete -f https://raw.githubusercontent.com/skydive-project/skydive-operator/master/deploy/role.yaml
+$ kubectl delete -f https://raw.githubusercontent.com/skydive-project/skydive-operator/master/deploy/service_account.yaml
+```
+
+## Configuration
+
+The netflowcollector custom resource can be customized using the following configuration parameters:
+
+| Parameter                            | Description                                     | Default                                                    |
+| ----------------------------------   | ---------------------------------------------   | ---------------------------------------------------------- |
+| `exporter.write.s3.endpoint`         | Endpoint of the Object Storage to be used       | `http://127.0.0.1:9000`                                    |
+| `exporter.write.s3.installLocalMinio`| Install default Minio Object Storage locally    | `true`                                                     |
+| `exporter.write.s3.region`           | Object Store region                             | `default`                                                  |
+| `exporter.write.s3.use_api_key`      | Use an api key for Object Store autentication   | `false`                                                    |
+| `exporter.write.s3.api_key`          | api key for Object Store autentication          | Empty                                                         |
+| `exporter.write.s3.access_key`       | access key for Object Store autentication       | `admin`                                                    |
+| `exporter.write.s3.secret_key`       | secret key for Object Store autentication       | `admin1234`                                                |
+| `exporter.store.bucket`              | bucket name to be used in Object Store          | `default`                                                  |
+| `exporter.store.objectPrefix`        | prefix of stroed objects                        | `default`                                                  |
+
+The Skydive custom resource can be castomized using the following configuration parameters (and generally using any of the helm configuration parameters defined[https://github.com/skydive-project/skydive-helm](https://github.com/skydive-project/skydive-helm).):
+
+| Parameter                            | Description                                     | Default                                                    |
+| ----------------------------------   | ---------------------------------------------   | ---------------------------------------------------------- |
+| `image.repository`                   | Skydive image repository                        | `ibmcom/skydive`                                           |
```

Comment:
default should be the community image

Reply:
This wrong value is copied from skydive-helm https://github.com/skydive-project/skydive-helm documentation.. (the actual value is skydive/skydive)

Reply:
nonetheless please correct